frontend/src/utils/mapLookupCode.ts

```diff
@@ -0,0 +1,20 @@
+import { ILookupCode } from 'actions/lookupActions';
+import { SelectOption } from 'components/common/form';
+
+/**
+ * Convert an ILookupCode into a SelectOption.
+ *
+ * @param {ILookupCode} code - The code value to identify this item.
+ * @param {(number | string | null)} [defaultId] - The default ID.
```

Comment:
what does the defaultId do? would prefer comments that indicate funtionality or intended use, or skip if self explanatory and.

Reply:
This was inherited functionality.  I simply wanted to keep consistent with intellisense.  Self explanatory doesn't limit itself to no comment.  Something obvious to one person isn't obvious to another.